### DISCUSSION TITLE
@Mutexman

### DISCUSSION THREAD

**Discussion by Mutexman** [gijzelaerr/python-snap7#328] [Q&A] [answered]
Is there a similar C ++ layer around snap7? 
@Mutexman 
1) This isue is closed.
2) Before you start opening a new issue, pls. use [Discussions](https://github.com/gijzelaerr/python-snap7/discussions) for questions (so we can seperate it)
3) This repo is a python layer around snap7 ...I don't know what you mean with "how to use this in C++"???

EDIT: Where do you got problems with https://github.com/gijzelaerr/python-snap7/blob/b6481ec45bb6652288e991d6d90594b6bf2c64d0/snap7/util.py#L114 ?

_Originally posted by @swamper123 in https://github.com/gijzelaerr/python-snap7/issues/51#issuecomment-979132867_

**Comment by gijzelaerr** [comment, marked as answer]
Closing -- this was created from a converted issue comment and has no actionable content.
